scss/_pagination.scss

```diff
@@ -30,7 +30,7 @@
 }
 
 .page-item {
-  @if $pagination-margin-left == - $pagination-border-width {
```

Comment:
I guess that, latest space fix doesn't match with linting rules

Reply:
Woops, I've updated this to brackets which looks a bit better